src/entries/web-compiler.js

```diff
@@ -16,6 +17,8 @@ const cache2: { [key: string]: CompiledFunctions } = Object.create(null)
 const baseOptions: CompilerOptions = {
   expectHTML: true,
   preserveWhitespace: true,
+  modules,
+  staticKeys: genStaticKeys(modules),
```

Comment:
If we do this the basic static keys `'type,tag,attrsList,attrsMap,plain,parent,children,staticAttrs'` also put into the entry file. That's not good from my opinion.